### DISCUSSION TITLE
Dear @WizardMantis441

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#352] [General]
I think it's time we stop setting the wiki aside and update it. I can't figure out this stuff on my own. Where do scripts go; what are character functions; what the hell is a DiscordPresenceUpdateEvent? So many questions, and the wiki is unfinished. I don't have discord or twitter (and, frankly, would pay to not make a twitter account), so if you could respond to this directly, that would be great if we could contact via direct email. I understand if that's a big ask, and I understand if you don't want to use email.

Best endeavors,
-- MK

**Comment by WizardMantis441** [comment]
oh damn sorry for not seeing this
1) there is an actually a wiki being developed and is getting super far, check https://fnf-cne-devs.github.io/
2) i did not know u needed to pay to make a twitter account i thought u just needed to connect an email lol

**Reply by r6915ee** [reply]
If I remember correctly, the latter is only in certain countries. But it's still a mess anyway.

**Reply by THEkatinamicrowave** [reply]
Tysm! Yeah it's totally fine. I didn't know there was a github.io wiki that's actually really useful thank you
Also I was saying that I would pay money if it meant that I *didn't* have to make a Twitter account (I hate twitter die die die die die)
Many thanks
